returnn/config.py

```diff
@@ -88,9 +93,11 @@ def __init__(
 
         :param dict config: dictionary of the RETURNN config variables that are hashed
         :param dict post_config: dictionary of the RETURNN config variables that are not hashed
-        :param None|dict[int, dict[str, Any]] staged_network_dict: dictionary of network dictionaries,
+        :param None|dict[int, str|dict[str, Any]] staged_network_dict:
+            dictionary of network dictionaries or
+            returnn_common generated strings (as str variable from get_ext_net_dict_py_code_str),
```

Comment:
this comment is misleading.. sure the change now is to incorporate returnn_common, but in principle any string that somehow contains `network = something_valid_that_defines_a_network` can be given here independent of returnn_common.

Maybe we can update this docstring along the lines "or a str containing valid python code that defines a variable named `network`" to make the interface more clear. 
Feel free to add a reference to `get_ext_net_dict_py_code_str` as an example source of such a string.